src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtins/gtir_builtin_select.py

```diff
@@ -0,0 +1,125 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from typing import Callable
+
+import dace
+
+from gt4py import eve
+from gt4py.next.iterator import ir as itir
+from gt4py.next.iterator.ir_utils import common_pattern_matcher as cpm
+from gt4py.next.program_processors.runners.dace_fieldview.gtir_builtins.gtir_builtin_translator import (
+    GTIRBuiltinTranslator,
+)
+from gt4py.next.program_processors.runners.dace_fieldview.utility import get_symbolic_expr
+from gt4py.next.type_system import type_specifications as ts
+
+
+class GTIRBuiltinSelect(GTIRBuiltinTranslator):
+    """Generates the dataflow subgraph for the `select` builtin function."""
+
+    true_br_builder: Callable
+    false_br_builder: Callable
```

Comment:
Just a question: isn't it possible to make these type hints more strict (e.g. defining the return type of the callables)?

Reply:
Right. I have introduced the type alias:
`SDFGFieldBuilder: TypeAlias = Callable[[], list[SDFGField]]`